ClientApp/viking-vault/src/UserDashboard/AccountPage.tsx

```diff
@@ -261,24 +261,22 @@ class AccountPage extends React.Component<any, IAccountPageState>{
 
     render(){
         return(
-            <div className="page-background">
-                <div className="account-view"> 
-                    { this.state.redirect? <Redirect to = "/login"  /> : null}       
-                    {this.state.accountInfo.isPresent ?  this.showUnavailableCard() : this.showCardInformations()}
-                    <div className="accounts-information">
-                        <div className="accounts-title">
-                            <h2 className="accounts-header">Accounts</h2>
-                        </div>
-                        {this.state.accountInfo.isPresent ? this.inexistentCardNotification() : this.accountsInformation()}
+            <div className="account-view"> 
+                { this.state.redirect? <Redirect to = "/login"  /> : null}       
```

Comment:
You have two `{this.state.redirect? <Redirect to="/login"/> : null}` in your `render()`, please remove one